README.md

```diff
@@ -97,7 +97,10 @@ plugins: [
 https://codepen.io/team/codepen/pen/PNaGbb
 ```
 
-#### Result
+<details>
+  <summary>
+  Output
```

Comment:
Can you make this `Result` again and make it bold too please?

Also put it on 1 line please.